### DISCUSSION TITLE
How to run related job(1:n)

### DISCUSSION THREAD

**Discussion by mqq-marek** [dagster-io/dagster#14702] [Q&A]
I need to run a job which control / continue execution of related jobs. 
I do this by starting a job with op which starts sub jobs(dynamic amount and config) and next in main job goes to another op when I wait in infinite loop for finishing all sub jobs and do some final tasks.

Is there any better approach for such  process in dagster?

**Comment by sryza** [comment]
If the number and config of the sub-jobs is based on what happens at runtime, then what you've described is probably the best approach.

Certain dynamic patterns can be handled with dynamic graphs: https://docs.dagster.io/concepts/ops-jobs-graphs/dynamic-graphs#dynamic-graphs
